testing/node_fetcher.ts

```diff
@@ -86,7 +87,7 @@ export async function nodeFetcher(options: BaseFetcherOptions): Promise<FetcherF
 
   const init: RequestInit = {
     method,
-    timeout,
+    signal: timeoutSignal(timeout!),
```

Comment:
As suggested by https://github.com/node-fetch/node-fetch/blob/HEAD/docs/v3-UPGRADE-GUIDE.md#the-timeout-option-was-removed